packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -1,44 +1,45 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
+import { useState, useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Button } from '@wordpress/components';
-import { LEFT, RIGHT, UP, DOWN, BACKSPACE, ENTER } from '@wordpress/keycodes';
+import { Button, Notice, withSpokenMessages } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { URLInput } from '../';
 
-const handleLinkControlOnKeyDown = ( event ) => {
-	const { keyCode } = event;
-
-	if ( [ LEFT, DOWN, RIGHT, UP, BACKSPACE, ENTER ].indexOf( keyCode ) > -1 ) {
-		// Stop the key event from propagating up to ObserveTyping.startTypingInTextField.
-		event.stopPropagation();
-	}
-};
-
-const handleLinkControlOnKeyPress = ( event ) => {
-	const { keyCode } = event;
-
-	event.stopPropagation();
-
-	if ( keyCode === ENTER ) {
-	}
-};
-
 const LinkControlSearchInput = ( {
 	value,
 	onChange,
 	onSelect,
 	renderSuggestions,
 	fetchSuggestions,
 	showInitialSuggestions,
+	errorMessage,
+	speak = noop,
 } ) => {
 	const [ selectedSuggestion, setSelectedSuggestion ] = useState();
 
+	useEffect( () => {
```

Comment:
Per https://github.com/WordPress/gutenberg/pull/19775#discussion_r377770629 and the mentioned #15745 merge, this and the `<div aria-live>` should be removed.

Reply:
Fixed. Thanks for noting this update got merged.